depmapomics/copynumbers.py

```diff
@@ -408,10 +410,10 @@ def _CCLEPostProcessing(
         )
         wessegments, genecn, wesfailed = postProcess(
             wesrefworkspace,
-            AllSamplesetName if AllSamplesetName else samplesetname,
+            setEntity="pair_set",
```

Comment:
let's make this a parameter of the _CCLEPostProcessing function instead (and a config.py constant)